metadome/presentation/web/templates/dashboard.html

```diff
@@ -16,6 +16,13 @@
 });
 </script>
 
+{% if transcript_id|length > 0 %}
```

Comment:
Is this necesarry here ? Otherwise please move to dashboard.js

Reply:
dashboard.js is called without any parameters. So it doesn't know whether the user calls "metadome/dashboard/" or "/metadome/dashboard/transcript/ENST00000311502.7/". So the script can't really tell what function it needs to call when it's loaded.

The html page however, does get the parameter "transcript_id". This allows the page to call the function "visualizeTranscript" ONLY when "transcript_id" is set.

If the call would be in dashboad.js, then the html page would need to call it with parameters in the url.

Reply:
okay, thanks